README.md

```diff
@@ -18,13 +18,22 @@ This plugin is used to create and setup either a B2B or a B2C store with data wi
 
 We always recommend using the latest version of these commands, however, you can install a specific version or tag if needed.
 
-[commerce-on-lightning-orchestration](https://github.com/forcedotcom/commerce-on-lightning-orchestration) - commerce-orchestration
-
--   scratchorgs
-
-[commerce-on-lightning](https://github.com/forcedotcom/commerce-on-lightning) - @salesforce/commerce <---- THIS PLUGIN
-
--   store
+-   [commerce-on-lightning](https://github.com/forcedotcom/commerce-on-lightning) - @salesforce/commerce <---- THIS PLUGIN
+    -   create stores
+    -   install command: `sfdx plugins:install @salesforce/commerce`
+    -   install specific version:`sfdx plugins:install @salesforce/commerce@242.0.26`
+
+#### Plugin Dependencies
+
+1. [commerce-on-lightning-orchestration](https://github.com/forcedotcom/commerce-on-lightning-orchestration) - commerce-orchestration
```

Comment:
this plugin is no longer a dependency. The features have been added to this (commerce-on-lightning) plugin

Reply:
thanks! removed as dependency